benchmarks.go

```diff
@@ -119,6 +119,23 @@ type BenchmarkCase struct {
 }
 
 var benchmarkCases = []BenchmarkCase{
+	{
+		Name: "fastape",
+		URL:  "github.com/nazarifard/fastape",
+		New:  fastape.NewTape,
+
+		UnsafeStringUnmarshal: true,
```

Comment:
If fastape has both safe and unsafe unmarshalling versions, you should have the `fastape_reuse` use the unsafe version, and this one use the safe string decoding version.